src/init.cpp

```diff
@@ -394,6 +394,7 @@ std::string HelpMessage(HelpMessageMode mode)
     }
     strUsage += HelpMessageOpt("-shrinkdebugfile", _("Shrink debug.log file on client startup (default: 1 when no -debug)"));
     strUsage += HelpMessageOpt("-testnet", _("Use the test network"));
+    strUsage += HelpMessageOpt("-requirestandard", _("Require transactions to be standard even in testnet/regtest"));
```

Comment:
To be clear, the way -requirestandard works is that it can only turn on IsStandard() - it can't turn it off. Turning off IsStandard() is problematic on mainnet due to DoS attacks, and this simplistic implementation doesn't have any code to prevent those DoS attacks. (like @luke-jr's -acceptnonstdtxn patch does)

With the above in mind, would you still say the -help text should have the default thing? I think that'd give the user the wrong impression that passing -requirestandard=0 in mainnet would turn off IsStandard()